### PR TITLE
Fetch polyfill

### PR DESCRIPTION
**Pull request by krichprollsch**

### CODE REVIEW

**Comment by krichprollsch** [comment]
follow-up: https://github.com/lightpanda-io/browser/issues/319

File: src/polyfill/fetch.zig [hunk 1]
```diff
@@ -0,0 +1,719 @@
...
+//
+// The original code source is available in MIT license.
+//
+// The script comes from the built version from npm.
+// You can get the package with the command:
+//
+// wget $(npm view whatwg-fetch dist.tarball)
+//
+// The source is the content of `package/dist/fetch.umd.js` file.
+pub const source =
```
Comment (francisbouvier) [inline src/polyfill/fetch.zig]: What about commit the js file directly and use the directive `@embedFile`?

Comment (francisbouvier) [reply src/polyfill/fetch.zig]: It will be easier to read and modify if needed.

File: tests/html/await-fetch.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
```
Comment (francisbouvier) [inline tests/html/await-fetch.html]: Where is this test is played?

File: src/polyfill/fetch.zig (hunk 1 above)
Comment (krichprollsch) [reply src/polyfill/fetch.zig]: I didn't know `@embedFile`, ty :pray:
It's much better to keep a separated pure js file.

File: tests/html/await-fetch.html (hunk 2 above)
Comment (krichprollsch) [reply tests/html/await-fetch.html]: I used the file for manual tests. Removed now.

**Pull request merged**